defaults/monitoring/.github/workflows/org-monitoring.yml

```diff
@@ -18,7 +18,6 @@
 # Doc & support: https://sfdx-hardis.cloudity.com/salesforce-monitoring-home/
 
 on:
-  push:
```

Comment:
The only time when the push is from you is when you configure the job, and it helps to see that the first run works :)
So please can you let the push event here ? :)

Reply:
This runs when you create the initial branch, as well as when you merge. There is an easy way of running actions with the `workflow_dispatch`. 

If you still want on the push event I can add branch specific events

Reply:
Hmmmm I forgot that GitHub case is different from other Git providers that allow scheduling from other branches than default one :)

So ok for me to remove push here :)